components/DropDown.js

```diff
@@ -7,16 +7,16 @@ export default class DropDown extends React.Component {
   items = () => {
     let items = [
       {
-        label: '1',
-        value: '1',
+        label: this.props.label1,
```

Comment:
Don't use numbers in variable names.

I don't think this is the best way to approach this array full of dictionaries. Instead of having six different props. You can just have one prop that should be an array. So essentially, you would define this `items` array outside of this class, and then pass it in as a prop for the `DropDown` component.